var/da/da_radiance/da_qc_ahi.inc

```diff
@@ -163,7 +164,7 @@ subroutine da_qc_ahi (it, i, nchan, ob, iv)
 
       if ( isflg > 0 ) then         
          do k = 1, nchan                
-            if ( k /= 2 .and. k /= 3 .and. k /= 4 ) then
+            if ( k /= 2 .and. k /= 3 .and. k /= 4 ) then ! This is questionable...
```

Comment:
channels 2-4 are water vapor channels, which are Ok to assimilate over land. So this check is for other window channels.